### ISSUE TITLE
Limiting the bot to certain-named (group)chats

### ISSUE THREAD

**Issue by asapabri** [Synthesized-Infinity/Whatsapp-Botto-Xre#45]
I wanted to ask if it was possible to adjust the code to add that kind of feature in an easy way (since I barely know Java basics)

**Comment by SomnathDas** [comment]
Yes you can do that, in easy way? I'm not sure what you meant by that. It's built on typescript/javascript not JAVA 

@asapabri

**Comment by asapabri** [comment]
I'm saying that I don't know anything about coding except some basics in Java Editor we learned at school...just asking if it was possible to watch a few tutorial and google stuff to add code for the adjustment I wanted because I myself couldn't set up the Bot via self hosting let alone code in a language I don't know (barely know since I guess Javascript is similar to Java)
